lib/maze/hooks/appium_hooks.rb

```diff
@@ -30,6 +31,11 @@ def after(scenario)
 
       def after_all
         @client&.log_run_outro
+        if $success
+          Maze::Plugins::DatadogMetricsPlugin.send_increment('appium.test_succeeded')
+        else
+          Maze::Plugins::DatadogMetricsPlugin.send_increment('appium.test_failed')
```

Comment:
I can't see that the device farm in use is being included?  We need to be able to distinguish between BB and BS (and I would not send any metrics at all for `:local`).